domray/env/domrl/agents/provincial_agent.py

```diff
@@ -75,103 +89,76 @@ def Sort_List_Of_Tuples(tup):
 
 # helper to check if a card is in their hand
 def hand_contains(state_view, card_name):
-    if card_name in state_view.player.hand:
-        return True
+    for card in state_view.player.hand:
+        if card == card_name:
+            return True
```

Comment:
This change isn't necessary; the original two lines is a more pythonic way of doing the same thing.